### PR TITLE
Improvements to man pages for systemd.service, sd-bus, and better error messages

### PR DESCRIPTION
**Pull request by keszybz**
Some more cleanup/fixup/documentation/error-message improvements split out of #27113.

### CODE REVIEW

File: src/core/execute.c [hunk 1]
```diff
@@ -4397,6 +4397,18 @@ static int collect_open_file_fds(
         return 0;
 }
 
+static void log_command_line(Unit *unit, const char *msg, const char *executable, char **argv) {
+        if (!DEBUG_LOGGING)
+                return;
+
+        _cleanup_free_ char *cmdline = quote_command_line(argv, SHELL_ESCAPE_EMPTY);
+
```
Comment (bluca) [inline src/core/execute.c:4409]: should have an `assert(unit)` as the macros just deref it

**Review by bluca — APPROVED** [review]

File: src/core/dbus-unit.c [hunk 2]
```diff
@@ -2407,7 +2407,7 @@ int bus_unit_set_properties(
         assert(u);
         assert(message);
 
-        /* We iterate through the array twice. First run we just check
+        /* We iterate through the array twice. First run just checks
          * if all passed data is valid, second run actually applies
          * it. This is to implement transaction-like behaviour without
          * actually providing full transactions. */
```
Comment (poettering) [inline src/core/dbus-unit.c]: if you touch this, maybe rebreak?

File: src/core/execute.c [hunk 3]
```diff
@@ -5197,9 +5196,10 @@ static int exec_child(
         }
 #endif
 
-        /* We repeat the fd closing here, to make sure that nothing is leaked from the PAM modules. Note that we are
-         * more aggressive this time since socket_fd and the netns and ipcns fds we don't need anymore. We do keep the exec_fd
-         * however if we have it as we want to keep it open until the final execve(). */
+        /* We repeat the fd closing here, to make sure that nothing is leaked from the PAM modules. Note that
+         * we are more aggressive this time, since we don't need socket_fd and the netns and ipcns fds any
+         * more. We do keep exec_fd however, if we have it, since we need to keep it open until the final
```
Comment (poettering) [inline src/core/execute.c:5217]: any more → anymore

Comment (poettering) [reply src/core/execute.c:5217]: ah, hmm, just looked this up, apparently that is an americanism, and optional, so ignore me

**Comment by keszybz** [comment]
Updated with the added `assert`s and reflowing of the text.

**Comment by keszybz** [comment]
jammy-amd64:
TEST-17-UDEV:                       FAIL     (1221 s)
TEST-29-PORTABLE:                   FAIL     (355 s)

jammy-ppc64el:
TEST-67-INTEGRITY:                  FAIL     (179 s)

Those seem to be just timeouts because of overload. The tests have been running for 6+ hours now.

**Pull request merged**